chroma_core/management/commands/print-settings.py

```diff
@@ -65,6 +79,7 @@ def handle(self, *args, **kwargs):
             "AMQP_BROKER_URL": settings.BROKER_URL,
             "BRANDING": settings.BRANDING,
             "USE_STRATAGEM": settings.USE_STRATAGEM,
+            "INCLUDES": settings.INCLUDES,
```

Comment:
BTW, with mustache it could be just a list of files (not a valid nginx statement, what it is as far as I know):

```
{{#includes}}
  include {{path}};
{{/includes}}
```